src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,14 @@
 import React from 'react';
 import './MoviesList.scss';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  movies: Movie[];
+};
+
+export const MoviesList: React.FC<Props> = ({ movies }) => (
+  <div className="movies">
+    {movies.map(movie => <MovieCard movie={movie} />)}
```

Comment:
you could add key to each <MovieCard>, code works without it but i believe we talked about why we should use them on today's lesson

Reply:
totally agree with Paweł, it's important to add key to every mapped element 😉 https://react.dev/learn/rendering-lists#keeping-list-items-in-order-with-key